test/client/app/components/siteList/publishedState.test.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import moment from 'moment'
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
 import PublishedState from '../../../../../assets/app/components/siteList/publishedState';
 
 const NO_BUILDS_MESSAGE = 'Please wait for build to complete or check logs for error message.';
 const PUBLISHED_BASE = 'Please wait for build to complete or check logs for error message.';
-const MOST_RECENT_BUILD = 'This site was last published at September 4th 2015, 3:11:23 pm.';
+const FORMATTED_MOST_RECENT_BUILD_TIME =  moment("2015-09-04T15:11:23.000Z").format('MMMM Do YYYY, h:mm:ss a')
```

Comment:
can you extract this string representing the build time as a constant to make it absolutely clear that this value matches the value a few lines later?